New_PlatformGame/Assets/PauseButtonScript.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.SymbolStore;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+public class PauseButtonScript : MonoBehaviour
+{
+    public GameObject pausePanel;
+    public GameObject returnButton;
+    public GameObject mainMenuButton;
+    public GameObject settingButton;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        GetComponent<Button>().onClick.AddListener(Pause);
+
+        returnButton.GetComponent<Button>().onClick.AddListener(Play);
+        settingButton.GetComponent<Button>().onClick.AddListener(Settings);
+        mainMenuButton.GetComponent<Button>().onClick.AddListener(GotoMainMenu);
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
```

Comment:
```suggestion
    // Update is called once per frame
    void Update()
    {

    }
```
작동 내용이 없는 Update 메소드는 필요할 때 까지는 없애둬도 상관없다. 그 편이 이쁘다.